tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher.py

```diff
@@ -330,27 +330,50 @@ def get_cpu_extension(cpu_extensions, selection_mode):
         if cpu_extensions_name != 'AUTO':
             return cpu_extensions
         extensions_path = cpu_extensions.parent
-        file_format = '{}.dll' if platform.system() == 'Windows' else 'lib{}.so'
-        if not selection_mode:
-            default_cpu_extension = file_format.format('cpu_extension')
-            extension_list = list(extensions_path.glob(default_cpu_extension))
-
-            if extension_list:
-                return extension_list[0]
-
-            cpu_info_flags = get_cpu_info()['flags']
-            supported_flags = ['avx512', 'avx2', 'sse4']
-            for flag in supported_flags:
-                selection_mode = flag
-                if selection_mode in cpu_info_flags:
-                    break
-        extension_list = list(extensions_path.glob(file_format.format('cpu_extension_{}'.format(selection_mode))))
+        os_specific_formats = {
+            'Darwin': 'lib{}.dylib',
```

Comment:
BTW, macOS loadable modules (plugins) [should have `.so` extension](https://gitlab.kitware.com/cmake/cmake/blob/v3.14.1/Modules/Platform/Darwin.cmake#L34-38) (internal bug has been filled for DLDT), so it make sense to add support for multiple variants.

Reply:
What should have higher priority `.so` or `dylib`?

Reply:
ok, I add multiple variants

Reply:
I don't think it matters. There shouldn't ever be both.